webhooks-extension/endpoints/webhook_test.go

```diff
@@ -194,13 +209,273 @@ func testGetAllWebhooks(expectedWebhooks []webhook, r *Resource, t *testing.T) {
 	}
 }
 
+func TestDeleteByNameNoName405(t *testing.T) {
+	setUpServer()
+	httpReq, _ := http.NewRequest(http.MethodDelete, server.URL+"/webhooks/?namespace=foo", nil)
+	response, _ := http.DefaultClient.Do(httpReq)
+	if response.StatusCode != 405 {
+		t.Errorf("Status code not set to 400 when deleting without a name, it's: %d", response.StatusCode)
+	}
+}
+
+func TestDeleteByNameNoNamespaceBadRequest(t *testing.T) {
+	setUpServer()
+	httpReq, _ := http.NewRequest(http.MethodDelete, server.URL+"/webhooks/name=foo", nil)
```

Comment:
I think `webhooks/name=foo` should be `webhooks/foo` because the name is a path parameter.